.github/workflows/nightly-ci-alpha-release.yml

```diff
@@ -0,0 +1,152 @@
+    runs-on: ubuntu-latest
+    steps:
+      # Check-out repo
+      - uses: actions/checkout@v2
+        with:
+          ref: feature/conditional-compilation
+      # Check if any changes have been pushed to branch since last release
+      - name: Check latest commit age
+        id: checkChange
+        # Here we query the github rest api for the commits, use jq (json parser) to grab the first commit and put contents into $HOME/commit.json
+        # Note: we ignore commits made by the bot account that commits the auto version bump changes.
+        # Then get the timestamp date for the first commit and check if it was over a day old.
+        run: |
+          curl -H 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' -sL https://api.github.com/repos/${{ github.repository }}/commits | jq -r '[.[] | select(.author.login != "github-action-bot")][0]' > $HOME/commit.json
+
+          commitdate=$(jq -r '.commit.author.date' $HOME/commit.json)
+          commiturl=$(jq -r '.url' $HOME/commit.json)
+          echo Last commit: $commiturl
+          echo Last commit date \(UTC\): $commitdate
+
+          timestamp=$(date --utc -d "$commitdate" +%s)
+          echo Commit timestamp: $timestamp
+
+          currentTimestamp=$(date --utc +%s)
+          echo Current timestamp: $currentTimestamp
+
+          days=$(( ($currentTimestamp - $timestamp ) / 86400 ))
+          echo Days since last commit: $days
+
+          if [ $days = "0" ]; then
+            echo "::set-output name=hasChanged::true"
+          else
+            echo "::set-output name=hasChanged::false"
+          fi
+
+  release:
+    name: Create alpha releases
+    needs: checkForChanges
+    if: needs.checkForChanges.outputs.hasChanged == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      # Check-out repo
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: feature/conditional-compilation
+
+      # Ensure node version is great enough
+      - name: Use Node.js v14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      # Try get node_modules from cache
+      - name: Restore node_modules from cache
+        uses: actions/cache@v2
+        with:
+          path: common/temp/pnpm-store
+          key: ${{ runner.os }}-${{ hashFiles('common/config/rush/pnpm-lock.yaml') }}
+
+      # Install dependencies
+      - name: Install rush
+        run: npm install -g @microsoft/rush@5.47.0
+      - name: Install dependencies
+        run: rush install
+
+      # Get datetime
+      - name: Get datetime for alpha release name
+        id: datetime
+        run: |
+          echo "::set-output name=datetime::$(date +'%Y%m%d.%H%M')"
+
+      # Bump alpha package versions
+      - name: Bump alpha package versions
+        run: node common/config/node_modules/beachball/bin/beachball canary --canary-name alpha-${{ steps.datetime.outputs.datetime }} --tag dev --no-publish
```

Comment:
Apply https://github.com/Azure/communication-ui-library/pull/1192 here (change `...alpha-...` to `...alpha+...`)